Gemfile

```diff
@@ -109,7 +109,7 @@ gem 'after_commit_action', '~> 1.1'
 gem 'aasm', '~> 5.2.0'
 gem 'active_scheduler', '~> 0.7.0'
 gem 'retriable', '~> 3.1'
-gem 'cld3', '~> 3.4.3'
+gem 'cld3', '= 3.5.0'
```

Comment:
@krbhavith Since this is an oddity, can you leave a comment here noting that higher patch versions caused deployment issues?  If you have a link noting the issue from the cld3 repo, might as well leave that here as well.  We want to leave record of why we are locking this for the time being to save us some future headaches.

Reply:
Updated comment, but could not find the exact issue on cld3 repo. I will post it on PR when if I find it.